tests/hardhat/Comptroller/Diamond/assetListTest.ts

```diff
@@ -1,38 +1,40 @@
 import { FakeContract, MockContract, smock } from "@defi-wonderland/smock";
 import { loadFixture, setBalance } from "@nomicfoundation/hardhat-network-helpers";
-import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import chai from "chai";
+import { Signer } from "ethers";
 import { ethers } from "hardhat";
 
-import { convertToUnit } from "../../../helpers/utils";
+import { convertToUnit } from "../../../../helpers/utils";
 import {
-  Comptroller,
   ComptrollerLens,
   ComptrollerLens__factory,
-  Comptroller__factory,
+  ComptrollerMock,
   IAccessControlManager,
   PriceOracle,
+  Unitroller,
   VBep20Immutable,
-} from "../../../typechain";
-import { ComptrollerErrorReporter } from "../util/Errors";
+} from "../../../../typechain";
+import { ComptrollerErrorReporter } from "../../util/Errors";
+import { deployDiamond } from "./scripts/deploy";
 
 const { expect } = chai;
 chai.use(smock.matchers);
 
 const { Error } = ComptrollerErrorReporter;
 
-describe("assetListTest", () => {
-  let root: SignerWithAddress; // eslint-disable-line @typescript-eslint/no-unused-vars
-  let customer: SignerWithAddress;
-  let comptroller: MockContract<Comptroller>;
+describe("Comptroller: assetListTest", () => {
+  let root: Signer; // eslint-disable-line @typescript-eslint/no-unused-vars
+  let customer: Signer;
```

Comment:
What's wrong with `SignerWithAddress` from hardhat-ethers though? I think `customer.address` is more readable than `await customer.getAddress()` 🤔

Reply:
Yes, we will change it in other tests wherever it's used.